dask/README.md

```diff
@@ -60,6 +60,7 @@ The following table lists the configurable parameters of the Dask chart and thei
 | Parameter                | Description             | Default        |
 | ------------------------ | ----------------------- | -------------- |
 | `scheduler.name` | Dask scheduler name. | `"scheduler"` |
+| `scheduler.component` |  | `"scheduler"` |
```

Comment:
I'm curious why these `component` options are configurable? What use cases do you see for changing them?

Reply:
Hi, mainly because the value 'scheduler/worker/jupyter' is too generic :-) Think of making dask a part of large system, there might be existing deployments using one of these component value already. In that case, people might prefer to change it into something like 'dask-scheduler' instead.
my $0.02

Reply:
Fair enough. I'm just on guard for config creep, especially for things that wont get used much. I haven't seen a need for this.

If you think this will come up for folks and isn't just a "what if" then I'm happy for it to go in.